net-mail/gnubiff/gnubiff-2.2.17-r3.ebuild

```
@@ -0,0 +1,47 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit autotools
+
+DESCRIPTION="A mail notification program"
+HOMEPAGE="http://gnubiff.sourceforge.net/"
+SRC_URI="https://downloads.sourceforge.net/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-3+-with-openssl-exception"
+SLOT="0"
+KEYWORDS="amd64 x86"
+IUSE="debug fam nls password"
+
+RDEPEND="
+	>=x11-libs/gtk+-3:3
+	>=gnome-base/libglade-2.3
+	dev-libs/popt
+	password? ( dev-libs/openssl:0= )
+	fam? ( virtual/fam )
+	x11-libs/libX11
+	x11-libs/pango
+	x11-libs/gdk-pixbuf
+"
```

can we sort this please?

```suggestion
RDEPEND="
	dev-libs/popt
	>=gnome-base/libglade-2.3
	x11-libs/gdk-pixbuf
	>=x11-libs/gtk+-3:3
	x11-libs/libX11
	x11-libs/pango
	fam? ( virtual/fam )
	password? ( dev-libs/openssl:0= )
"

---

I admit I'm more concerned about the warning that libglade is deprecated, but without going in and re-engineering the code I don't see a fix for that.